docs/docs/creating-a-sitemap.md

```diff
@@ -2,7 +2,29 @@
 title: Creating a sitemap
 ---
 
-This is a stub. Help our community expand it.
+### What is a Site Map?
 
-Please use the [Gatsby Style Guide](/docs/gatsby-style-guide/) to ensure your
-pull request gets accepted.
+An [XML sitemap](https://support.google.com/webmasters/answer/156184?hl=en) lists a website’s important pages, making sure search engines (such as Google) can find and crawl them all, and helping it understand your website structure.
+
+Think of it as a map for your website. It shows what all of the pages are on your website.
+
+### Install
+
+To generate an XML sitemap, you will use the [`gatsby-plugin-sitemap`](/packages/gatsby-plugin-sitemap/) package. To install this package, run the following line on your command line:
+`npm install --save gatsby-plugin-sitemap`
+
+### How to use [gatsby-plugin-sitemap](/packages/gatsby-plugin-sitemap/)
```

Comment:
Perhaps transition into this section? e.g.

```md
Once installed, we can now add this plugin to our `gatsby-config.js`, like so:
```

Reply:
Good idea! I will reflect something like that on the next commit